src/content/ueber-uns/abteilungen.svx

```diff
@@ -1,57 +1,49 @@
 ---
 title: Abteilungen
-date: '2022-11-15'
+date: '2023-03-07'
 nav_index: 2
 ---
 
-## Allgemeines
+Die Task Force Hellcat besteht aus drei Abteilungen, die verschiedene Truppengattungen umfassen. Auf dieser Seite werden die Truppengattungen der Abteilungen näher erläutert.
 
-Die Task Force Hellcat ist in drei Abteilungen untergliedert, die jeweils durch einen Abteilungsleiter organisiert und verwaltet werden. Alle Abteilungen umfassen mehrere Truppengattungen, die auf dieser Seite erläutert werden.
+(TODO: Insert tfhc structure diagram here?)
 
-## Streitkräfte
+## Kampftruppen
 
-Die Abteilung der Streitkräfte wird von Leon organisiert und verwaltet. Sie umfasst insgesamt vier Truppengattungen: Die (motorisierte) Infanterie, den Sanitätsdienst und die Aufklärung.
+### Infanterie
 
-### (Motorisierte) Infanterie
+Die Truppengattung der Infanterie wird von Leon organisiert und verwaltet. Sie umfasst mit Handwaffen ausgerüstete, sich zu Fuß bewegende und kämpfende Soldaten, die in [Trupps](../ausbildung/truppenteile#trupp) oder [Gruppen](../ausbildung/truppenteile#gruppe) agieren. Zur Infanterie zählen die Rollen des [Schützen](../streitkraefte/schuetze), des [MG-Schützen](../streitkraefte/mg-schuetze), des [PA-Schützen](../streitkraefte/pa-schuetze) und des [Präzisionsschützen](../streitkraefte/praezisionsschuetze).
 
-Die Infanterie setzt sich aus Schützen, MG-Schützen, AT-Schützen, Präzisionsschützen zusammen. Ihre Bewegung und der Feuerkampf erfolgen ausschließlich zu Fuß. Alternativ können sich die Soldaten auf Fahrzeugen aufgesessen bewegen, auch motorisierte Infanterie genannt. Die Infanterie kann vielseitig eingesetzt werden. Sie erweist sich besonders im urbanen Gebiet und unwegsamen Gelände als effektiv, da Fahrzeuge dort nur eingeschränkt beweglich sind.
+_Bemerkung_: Infanterie, die aufgesessen auf [geschützten Fahrzeugen (DEAD)](../fuhrpark/geschuetzte-fahrzeuge) agiert, nennt man **motorisierte Infanterie**.
```

Comment:
I'd suggest using strikethrough to mark dead links, looks a little bit cleaner imo.

```suggestion
_Bemerkung_: Infanterie, die aufgesessen auf [~~geschützten Fahrzeugen~~](../fuhrpark/geschuetzte-fahrzeuge) agiert, nennt man **motorisierte Infanterie**.
```

You could use searching for `~~` in your editor to find links that need curating later.

Reply:
Resolved as of 48e52552c4ec8c8c2f617f947eaac8b529de6891.